token/client/src/token.rs

```diff
@@ -894,6 +916,88 @@ where
         .await
     }
 
+    /// Wrap lamports into native account
+    pub async fn wrap<S: Signers>(
+        &self,
+        account: &Pubkey,
+        owner: &Pubkey,
+        lamports: u64,
+        signing_keypairs: &S,
+    ) -> TokenResult<T::Output> {
+        // mutable owner for Tokenkeg, immutable otherwise
+        let immutable_owner = self.program_id != spl_token::id();
+        let instructions = self.wrap_ixs(account, owner, lamports, immutable_owner)?;
+
+        self.process_ixs(&instructions, signing_keypairs).await
+    }
```

Comment:
i liked the idea of making wrapped accounts immutable by default and exposing a second "you asked for it, you got it" function, in favor of taking a bool for that, or making mutable owner the default, but id be open to either of those interfaces if you prefer one